flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/HiveDialectITCase.java

```diff
@@ -940,6 +940,100 @@ public void testUnsupportedOperation() {
         }
     }
 
+    @Test
+    public void testInsertIntoNotExistStaticPartitionWithoutData() throws Exception {
```

Comment:
I think these tests can move to `HiveTableSinkITCase` and can be composed to one single test.